gradle/libs.versions.toml

```diff
@@ -63,6 +63,8 @@ firebaseAuthKtx = "23.0.0"
 firebaseDatabaseKtx = "21.0.0"
 firebaseFirestore = "25.1.0"
 firebaseUiAuth = "8.0.0"
+junitKtx = "1.2.1"
+uiTestJunit4AndroidVersion = "1.7.3"
```

Comment:
Conflict with main branch: Unresolved version conflict between `junitKtx`, `uiTestJunit4AndroidVersion`, and `navigationRuntimeKtx` as well as calendar library versions (`ical4j`, `compose`). Ensure all necessary versions for testing (`junitKtx`, `uiTestJunit4AndroidVersion`), navigation (`navigationRuntimeKtx`), and calendar libraries (`ical4j`, `compose`) are merged correctly without overriding each other.